protocol/messenger_settings.go

```diff
@@ -20,10 +20,18 @@ func (m *Messenger) SetLogLevel(request *requests.SetLogLevel) error {
 	return nodecfg.SetLogLevel(m.database, request.LogLevel)
 }
 
+func (m *Messenger) SetMaxLogBackups(request *requests.SetMaxLogBackups) error {
+	return nodecfg.SetMaxLogBackups(m.database, request.MaxLogBackups)
+}
+
 func (m *Messenger) SetCustomNodes(request *requests.SetCustomNodes) error {
 	return nodecfg.SetWakuV2CustomNodes(m.database, request.CustomNodes)
 }
 
+func (m *Messenger) SaveNewWakuNode(request *requests.SaveNewWakuNode) error {
+	return nodecfg.SaveNewWakuNode(m.database, request.NodeAddress)
```

Comment:
Same thing here

Reply:
Added a fix. 

I think it's a general check and we shouldn't be testing these here:
```
peer.AddrInfoFromP2pAddr
addr.ValueForProtocol
```